build_system/dune

```diff
@@ -16,9 +16,20 @@
   ANSITerminal)
  (modules clerk_driver))
 
+(rule
```

Comment:
What's that for ?

Reply:
Avoid a failure when the `custom_linking.sexp` file doesn't exist (we default it to a file containing just `()`)

Reply:
(we could also statically add the file to the source tree; the upside of that would be that it is harder to get confused because you had a stale file sitting there; but it's not very pretty)

Reply:
Ok and the `custom_linking.sexp` is produced by the `build_release.sh` script. Maybe that would be worth a comment in the dune file. I'm in favor of not adding `custom_linking.sexp` to the build tree.